601-spring-data-primitive-types/src/main/java/io/quarkus/qe/spring/data/BookRepository.java

```diff
@@ -1,12 +1,14 @@
 package io.quarkus.qe.spring.data;
 
-import io.quarkus.qe.spring.data.model.Book;
 import java.util.List;
+
 import org.springframework.data.jpa.repository.Query;
-import org.springframework.data.repository.Repository;
+import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.query.Param;
 
-public interface BookRepository extends Repository<Book, Integer> {
+import io.quarkus.qe.spring.data.model.Book;
+
+public interface BookRepository extends CrudRepository<Book, Integer> {
 
     // issue 9192
```

Comment:
Seems like the issue is gone, please update that class. The same goes for QUARKUS-525, search and remove respective comments in classes

Reply:
Well I think, that keep the reference issue there in the code it's important in order to understand why is this code there. But is true that we should avoid internal references as `QUARKUS-525` and keep public references. So I've replaced, `QUARKUS-525` by `issue 13015`

Reply:
Change it to: // issue 9192; resolved; regression test

Reply:
Please, add "TODO" and the fully qualified url of the issue, this way is much easier to search.

Reply:
@Sgitario as I understand this issue is already solved and is used to see regression. I guess there is no need for TODO, do you agree? I agree that a fully qualified name is better

Reply:
If the issue is already solved, I would avoid adding any links at all.